WordPressKit/Domains/DomainsServiceRemote+AllDomains.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+
+extension DomainsServiceRemote {
+
+    // MARK: - API
+
+    public func fetchAllDomains(params: AllDomainsEndpointParams? = nil, completion: @escaping (AllDomainsEndpointResult) -> Void) {
+        let path = self.allDomainsEndpointPath(params: params)
+        self.wordPressComRestApi.GET(path, parameters: nil) { result, _ in
```

Comment:
Have you considered relying on the `parameters` argument of the `GET(_, parameters:)` API client to build the URL query, as opposed to implement the logic manually here?

Reply:
@mokagio Good point, I didn't pay attention to that `parameters` argument. But I agree it should be the one to use.

Reply:
Resolved in 9d0a555a91d0db7a158f8bc613176a8be085180c